proto/storage/image_v2.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package storage;
+
+import "google/protobuf/timestamp.proto";
+import "storage/image.proto";
+
+option go_package = "./storage;storage";
+option java_package = "io.stackrox.proto.storage";
+
+// Next tag: 28
+message ImageV2 {
+  string id = 1; // @gotags: search:"Image ID,hidden" sql:"pk,type(uuid)"
+  string sha = 2; // @gotags: search:"Image Sha"
+  ImageName name = 3;
+
+  ImageMetadata metadata = 4;
+  ImageScan scan = 5; // @gotags: policy:"Image Scan"
+
+  ImageSignatureVerificationData signature_verification_data = 6;
+  ImageSignature signature = 7;
+
+  oneof set_components {
+    int32 components = 8; // @gotags: search:"Component Count,hidden"
+  }
+  oneof set_cves {
+    int32 cves = 9; // @gotags: search:"Image CVE Count"
+  }
+  oneof set_fixable_cves {
+    int32 fixable_cves = 10; // @gotags: search:"Fixable CVE Count,hidden"
+  }
+
+  oneof set_unknown_cves {
+    int32 unknown_cves = 11; // @gotags: search:"Unknown CVE Count,hidden"
+  }
+  oneof set_fixable_unknown_cves {
+    int32 fixable_unknown_cves = 12; // @gotags: search:"Fixable Unknown CVE Count,hidden"
+  }
+
+  oneof set_critical_cves {
+    int32 critical_cves = 13; // @gotags: search:"Critical CVE Count,hidden"
+  }
+  oneof set_fixable_critical_cves {
+    int32 fixable_critical_cves = 14; // @gotags: search:"Fixable Critical CVE Count,hidden"
+  }
+
+  oneof set_important_cves {
+    int32 important_cves = 15; // @gotags: search:"Important CVE Count,hidden"
```

Comment:
Graphql will return null values for cve count by severity in case there is no value returned by db?

Reply:
nil and 0 seem no different, but I'm sure someone will tell me the semantic difference therein that never really ever matters in the world